src/Application/Actions/Hooks/StripeChargeUpdate.php

```diff
@@ -182,6 +189,18 @@ private function handleChargeDisputeClosed(Event $event, Response $response): Re
             return $this->respond($response, new ActionPayload(204));
         }
 
+        if ($donation->getAmountFractionalIncTip() < $dispute->amount) {
```

Comment:
minor point but since we have more than in the comments imho it's easier a bit easier to follow if we use the more than symbol in the code too?
```suggestion
        if ($dispute->amount > $donation->getAmountFractionalIncTip()) {
```